src/alto-ui/Dropdown/Dropdown.scss

```diff
@@ -5,6 +5,13 @@
   // abtitary amount
   max-width: 55rem;
 
+  &--mediumWidth {
+    max-width: 45rem;
+  }
+  &--narrow {
+    max-width: 15rem;
+  }
```

Comment:
who decided that ?

Reply:
I took this value from git history. It can be useful in future